lib/elixir/lib/enum.ex

```diff
@@ -1413,15 +1413,18 @@ defmodule Enum do
       6
 
   """
+  @spec sum(Range.t) :: integer
```

Comment:
Unfortunately we can't, it brings a dialyzer warning.
Tereis one of the fix commits resulted from the linked issue: https://github.com/elixir-lang/elixir/pull/4432/commits/ee66a9d1096f29c513878ecac8162684b6a1f533.